From 0b2458d02ebd3bd27f86833ccf6b26d93e1981c8 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Sat, 9 Apr 2022 06:27:50 +0000
Subject: [PATCH 69/70] WIP: dt-bindings: arm: amlogic: add S905L and Venz V10
 bindings

Add SOC bindings for S905L devices and the board binding for Venz V10.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 Documentation/devicetree/bindings/arm/amlogic.yaml | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/Documentation/devicetree/bindings/arm/amlogic.yaml b/Documentation/devicetree/bindings/arm/amlogic.yaml
index 8f86ab3386c4..a6e5eabd1539 100644
--- a/Documentation/devicetree/bindings/arm/amlogic.yaml
+++ b/Documentation/devicetree/bindings/arm/amlogic.yaml
@@ -83,6 +83,13 @@ properties:
           - const: amlogic,s805x
           - const: amlogic,meson-gxl
 
+      - description: Boards with the Amlogic Meson GXL S905L SoC
+        items:
+          - enum:
+              - venz,v10
+          - const: amlogic,s905l
+          - const: amlogic,meson-gxl
+
       - description: Boards with the Amlogic Meson GXL S905W SoC
         items:
           - enum:
-- 
2.17.1

